### PR TITLE
Add example for custom OAuth profile configuration for Google

### PR DESCRIPTION
**Pull request by Cassie205**

### CODE REVIEW

**Review by dkocher — CHANGES_REQUESTED** [review]

File: tutorials/custom_oauth_client_id.md [hunk 1]
```diff
@@ -98,24 +98,40 @@ Download the [template](../protocols/profiles/google_client_id.md#sample-google-
 - {download}`Google Storage Custom OAuth Client ID.cyberduckprofile<../protocols/profiles/_static/Google Storage Custom OAuth Client ID.cyberduckprofile>`
 
 
-Use the _OAuth Client ID_ created to edit the `OAuth Client ID` and `OAuth Redirect Url` in the template connection profile leaving other keys unchanged.
+Use the _OAuth Client ID_ created to edit the `OAuth Client ID`, `OAuth Client Secret`, and `OAuth Redirect Url` in the template connection profile leaving other keys unchanged.
```
[the remark this answers is not in the document]
Comment (Cassie205) [reply tutorials/custom_oauth_client_id.md:101]: Sure? It is listed here: https://docs.cyberduck.io/protocols/profiles/google_client_id/#sample-google-drive-with-custom-oauth-client-id-connection-profile

[the remark this answers is not in the document]
Comment (dkocher) [reply tutorials/custom_oauth_client_id.md:101]: As _Optional_.

[the remark this answers is not in the document]
Comment (dkocher) [reply tutorials/custom_oauth_client_id.md]: Remove from the sample profile as not required.

[the remark this answers is not in the document]
Comment (Cassie205) [reply tutorials/custom_oauth_client_id.md]: We have some customers confused by the prompt receiving if the secret is not added in the profile therefor I'd like to keep it in the sample

File: tutorials/custom_oauth_client_id.md [hunk 2]
```diff
@@ -98,24 +98,40 @@ Download the [template](../protocols/profiles/google_client_id.md#sample-google-
 - {download}`Google Storage Custom OAuth Client ID.cyberduckprofile<../protocols/profiles/_static/Google Storage Custom OAuth Client ID.cyberduckprofile>`
 
 
-Use the _OAuth Client ID_ created to edit the `OAuth Client ID` and `OAuth Redirect Url` in the template connection profile leaving other keys unchanged.
+Use the _OAuth Client ID_ created to edit the `OAuth Client ID`, `OAuth Redirect Url`, and optional the `OAuth Client Secret` in the template connection profile leaving other keys unchanged.
```
Comment (dkocher) [inline tutorials/custom_oauth_client_id.md:101]: - No comma before `and`.
- `optional`→ `optionally`

File: tutorials/custom_oauth_client_id.md [hunk 3]
```diff
@@ -98,24 +98,40 @@ Download the [template](../protocols/profiles/google_client_id.md#sample-google-
...
-- `OAuth Client ID`. Override the registered application OAuth Client ID.
 - `OAuth Redirect Url`. Use the reverse notation of the OAuth Client ID and append `:oauth` to it.
+- `OAuth Client ID`. Override the registered application OAuth Client ID.
+
+::::
+::::{tab} Example
+
+Based on the credentials shown in step 11 the correct configuration would look like this:
+
+:::{image} _images/Edit_Custom_Client_ID_Profile.png
```
Comment (dkocher) [inline tutorials/custom_oauth_client_id.md:129]: Provide sample profile as editable code snippet instead of screenshot.

**Review by dkocher — CHANGES_REQUESTED** [review]

File: tutorials/custom_oauth_client_id.md (hunk 2 above)
Comment (Cassie205) [reply tutorials/custom_oauth_client_id.md]: Based on Grammarly, the comma before the `and` there belongs there because of the enumeration

**Review by dkocher — APPROVED** [review]

**Pull request merged**